SU-SWS/stanford_courses
# Hsdo 919 course academic year

# READY FOR REVIEW

# Summary
- Created a date field that sets the academic year based on what the selects from the "Academic Year" select list.
- Uses rules to compute the AY
- Hides the AY computed from user manipulation.
- This allows for easier filtering in views for current academic year (now + 153 days with a year specification)

# Needed By (Date)
- End of sprint

# Urgency
- Low

# Steps to Test

1. Checkout branch.
2. create/verify courses have an course section with an academic year.
3. `drush updb -y`
4. using views or devel, validate the academic year computed field is populated.

# Associated Issues and/or People
- https://github.com/SU-SWS/linky_clicky/pull/173

# See Also
- [Linky Clicky](https://github.com/SU-SWS/linky_clicky/pull/173)
